LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownstreamSender.cs

```diff
@@ -33,25 +33,24 @@ internal class DownstreamSender : IPacketForwarder
             this.logger = logger;
         }
 
-        public async Task SendDownstreamAsync(DownlinkPktFwdMessage message)
+        public async Task SendDownstreamAsync(DownlinkBasicsStationMessage message)
         {
             if (message is null) throw new ArgumentNullException(nameof(message));
             if (message.StationEui == default) throw new ArgumentException($"A proper StationEui needs to be set. Received '{message.StationEui}'.");
 
             if (this.socketWriterRegistry.TryGetHandle(message.StationEui, out var webSocketWriterHandle))
             {
-                var region = await this.basicsStationConfigurationService.GetRegionAsync(message.StationEui, CancellationToken.None);
-                var payload = Message(message, region);
-                LogSendingMessage(this.logger, message.StationEui, message.Txpk.Data, null);
+                var payload = Message(message);
+                LogSendingMessage(this.logger, message.StationEui, ConversionHelper.ByteArrayToString(message.Data), null);
```

Comment:
Probably there is too much logs, we could merge tis one with the log L76 on the same file